src/navigation.fs

```diff
@@ -32,8 +33,7 @@ module Navigation =
     /// Push new location into history and navigate there
     let newUrl (newUrl:string):Cmd<_> =
         [fun _ -> history.pushState((), "", newUrl)
-                  let ev = document.createEvent_CustomEvent()
-                  ev.initCustomEvent (NavigatedEvent, true, true, obj())
+                  let ev = CustomEvent.Create(NavigatedEvent)
```

Comment:
It's not recommended to use `initEvent` and the like anymore so I've changed it: https://developer.mozilla.org/en-US/docs/Web/API/Document/createEvent